edx-olive/edx-platform
# fix: removes duplicate platform name hard coded into the translation

## Description

Campus [noticed](https://lnet-jira.atlassian.net/browse/CAM2-487?focusedCommentId=30177) a repetition in the campus platform name for the "activate your account" sidebar notice.

This pull request addresses that issue.

## Supporting information

* **JIRA Ticket**: SE-3703, CAM2-487

## Testing instructions

* Create a new account
* Once you create a new account, you should be on the dashboard page, with side notifications similar to the ones shown below.
![image](https://user-images.githubusercontent.com/5631091/119797608-0f640980-bee3-11eb-938c-4986d77ac84c.png)
* From there, switch the language to Hebrew, in the navigation header.
* Once that's done, check the first sidebar notification, similar to the one shown below.
![image](https://user-images.githubusercontent.com/5631091/119797812-3b7f8a80-bee3-11eb-8a70-3cb1dab1a70d.png)
* The updated translation is in that sidebar notification.
![image](https://user-images.githubusercontent.com/5631091/119797880-50f4b480-bee3-11eb-87b1-1aabf5dd713f.png)


#### Before

![image](https://user-images.githubusercontent.com/5631091/119293437-17117d00-bc5b-11eb-8f2d-21c9988cad6c.png)

#### After

![image](https://user-images.githubusercontent.com/5631091/119293632-81c2b880-bc5b-11eb-918a-8d67a717e790.png)
